rpcs3/Emu/Cell/lv2/sys_usbd.cpp

```diff
@@ -290,6 +293,21 @@ usb_handler_thread::usb_handler_thread()
 
 	libusb_free_device_list(list, 1);
 
+	for (int i = 0; i < 8; i++) // Add VFS USB mass storage devices (/dev_usbXXX) to the USB device list
+	{
+		const cfg::device_info device = g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, fmt::format("/dev_usb%03d", i));
+
+		auto string_to_hex = [](const std::string& str) -> u16
```

Comment:
Can't you just add this as a function to the device_info struct?

Reply:
Sure thing!